cli/src/main/resources/nls/Help.properties

```diff
@@ -79,7 +79,7 @@ cmd.pgadmin.detail=pgAdmin is an open source management tool for PostgreSQL and
 cmd.quarkus=Tool commandlet for Quarkus (framework for cloud-native apps).
 cmd.quarkus.detail=Quarkus is a Kubernetes-native Java framework for building cloud-native applications. Detailed documentation can be found at https://quarkus.io/
 cmd.repository=Setup pre-configured git repositories (clone, build, import).
-cmd.repository.detail=TODO repository
+cmd.repository.detail=This will set up all pre-configured repositories. Simply call 'ide repository setup <your_project_name>', replace <your_project_name> with the name of your project config file located in 'settings/repository/your_project_name' and IDEasy will automatically clone, build and set up your project based on the existing property file.\nIf you omit the project name, all projects found in the repository directory will be pre-configured.
```

Comment:
```
Usage: ide [option]* repository setup [<repository>]
Setup pre-configured git repositories (clone, build, import).
```
```suggestion
cmd.repository.detail=Without further arguments this will set up all pre-configured git repositories.\nAlso, you can provide an explicit git repo as `<repository>` argument and IDEasy will automatically clone, build and set up your project based on the existing property file.\nRepositories are configured in 'settings/repository/<repository>.properties' and can therefore be shared with your project team for automatic or optional setup.
```